app/assets/stylesheets/homepage.css.scss

```diff
@@ -157,7 +155,7 @@
 
 
 .icon-parachute{
-  color: whitesmoke;
+  color: lightgray;
```

Comment:
Color `lightgray` should be written in hexadecimal form as `#d3d3d3`<br>Color literals like `lightgray` should only be used in variable declarations; they should be referred to via variable everywhere else.